core/os/input_event.cpp

```diff
@@ -311,10 +311,11 @@ bool InputEventKey::action_match(const Ref<InputEvent> &p_event, bool *p_pressed
 
 	bool match = get_scancode() == key->get_scancode() && (!key->is_pressed() || (code & event_code) == code);
 	if (match) {
-		if (p_pressed != NULL)
+		if (p_pressed != NULL) {
 			*p_pressed = key->is_pressed();
-		if (p_strength != NULL)
-			*p_strength = (*p_pressed) ? 1.0f : 0.0f;
+			if (p_strength != NULL)
```

Comment:
If `p_strength` is passed by reference and non NULL, it *should* be set, even if `p_pressed` is NULL.

BTW, both should be renamed `r_pressed` and `r_strength` to match our convention, as they are passed by reference (but that's not necessarily something to do in this PR).